fedimint-server/src/config/mod.rs

```diff
@@ -578,7 +578,7 @@ impl ServerConfig {
 /// The types of keys to run distributed key generation for
 #[derive(Clone, Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
 pub enum KeyType {
-    Hbbft,
+    Bft,
```

Comment:
Is this safe to rename? Just want to confirm since the enum implements `serde::Serialize` and `serde::Deserialize`.

Reply:
As far as I can tell we only use the serialized value during setup, so it's ok to break compatibility (we don't running the setup with different versions).